packages/backend-app-api/src/services/implementations/scheduler/schedulerServiceFactory.ts

```diff
@@ -20,7 +20,10 @@ import {
 } from '@backstage/backend-plugin-api';
 import { TaskScheduler } from '@backstage/backend-tasks';
```

Comment:
for now, this package retains the dependency on `backend-tasks` ... it's tempting to re-export from its new home instead, but because `backend-defaults` still depends on `backend-app-api`, i'm worried about introducing cyclical dependencies :)

Reply:
👍 , let's get everything in place and then consider options imo